dotcms-integration/src/test/java/com/dotcms/ai/viewtool/AiTest.java

```diff
@@ -84,6 +84,13 @@ private static Map<String, Secret> appConfigMap(final WireMockServer wireMockSer
                 Secret.builder().withType(Type.STRING).withValue(IMAGE_MODEL.toCharArray()).build(),
 
                 AppKeys.IMAGE_SIZE.key,
-                Secret.builder().withType(Type.SELECT).withValue(IMAGE_SIZE.toCharArray()).build());
+                Secret.builder().withType(Type.SELECT).withValue(IMAGE_SIZE.toCharArray()).build(),
+
+                AppKeys.LISTENER_INDEXER.key,
+                Secret.builder()
+                        .withType(Type.STRING)
+                        .withValue("{\"default\":\"blog\"}".toCharArray())
+//                        .withValue("{\"default\":\"blog\",\"blogsOnly\":\"blog.body\"}".toCharArray())
+                        .build());
```

Comment:
Why is this part of this PR?

Reply:
Needed to provide the listener configuration. Removing commented code.